src/main/java/carpetextra/mixins/BlockItemMixin_accurateBlockPlacement.java

```diff
@@ -13,11 +13,11 @@
 @Mixin(BlockItem.class)
 public class BlockItemMixin_accurateBlockPlacement
 {
-    @Redirect(method = "getPlacementState", at = @At(
-            value = "INVOKE",
-            target = "Lnet/minecraft/block/Block;getPlacementState(Lnet/minecraft/item/ItemPlacementContext;)Lnet/minecraft/block/BlockState;"
-    ))
-    private BlockState getAlternatePlacement(Block block, ItemPlacementContext itemPlacementContext_1)
+    @Shadow
+    private Block block //It's deprecated,but it's convenience
```

Comment:
You are missing a `;` here, also i believe it needs to be `@Shadow @Final @Deprecated private Block block;`